src/Microsoft.DotNet.VersionTools/Util/Command.cs

```diff
@@ -0,0 +1,349 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.DotNet.VersionTools.Util
+{
+    class Command
```

Comment:
@ericstj - do you have a strong opinion here?  The script needs to kick off a handful of processes (it looks like all 'git' commands).  Having an internal helper makes the code easier to read in those places.